server/utils/placements/index.test.ts

```diff
@@ -197,11 +197,11 @@ describe('placementUtils', () => {
           },
           {
             key: { text: 'Actual arrival date' },
-            value: { text: DateFormats.isoDateToUIDate(placement.actualArrivalDate) },
+            value: { text: DateFormats.isoDateToUIDate(placement.actualArrivalDateOnly) },
           },
           {
             key: { text: 'Arrival time' },
-            value: { text: DateFormats.timeFromDate(DateFormats.isoToDateObj(placement.actualArrivalDate)) },
+            value: { text: (placement.actualArrivalTime || '').substring(0, 5) },
```

Comment:
Is this `substring` needed here? The factory already returns time in 24hr format, I think? 🤔

Reply:
Yes, you're right. The API currently includes seconds so I originally made the factory do likewise.  Not any more though.